src/components/Image/RectangleImage/RectangleImage.module.scss

```
@@ -0,0 +1,23 @@
+@import "../../../styles/mixin_var.scss";
```

```suggestion
@import "/src/styles/mixin_var.scss";
```

`../`가 중복되는 경우, 절대 경로 사용을 고려해봐도 좋을 것 같아요!